backend/vkt/src/main/java/fi/oph/vkt/service/PublicEnrollmentEmailService.java

```diff
@@ -31,20 +32,23 @@ public class PublicEnrollmentEmailService {
   private final ReceiptRenderer receiptRenderer;
   private final TemplateRenderer templateRenderer;
 
+  @Value("${app.email.sending-enabled:false}")
+  private Boolean emailSendingEnabled;
+
   @Transactional
-  public void sendEnrollmentConfirmationEmail(final Enrollment enrollment, final Person person)
-    throws IOException, InterruptedException {
+  public void sendEnrollmentConfirmationEmail(final Enrollment enrollment) throws IOException, InterruptedException {
+    final Person person = enrollment.getPerson();
     final Map<String, Object> templateParams = getEmailParams(enrollment);
 
     final String recipientName = person.getFirstName() + " " + person.getLastName();
     final String recipientAddress = enrollment.getEmail();
     final String subject = "Vahvistus tutkintoon ilmoittautumisesta | Samma på svenska";
     final String body = templateRenderer.renderEnrollmentConfirmationEmailBody(templateParams);
 
-    final List<EmailAttachmentData> attachments = List.of(
-      createReceiptAttachment(enrollment, Language.FI),
-      createReceiptAttachment(enrollment, Language.SV)
-    );
+    // For local development
+    final List<EmailAttachmentData> attachments = emailSendingEnabled != null && !emailSendingEnabled
```

Comment:
Nyt jos `emailSendingEnabled == null` tulee tähän liitteet mukaan. Olisko parempi jos liitteet tulee vain jos tuo ei ole null ja on true?

Reply:
Taisin varuilta laittaa näin, että jos tuota ei ole määritelty niin logiikka toimii samalla tavalla kuin ennenkin. Eli, että liitteet tulee. Nyt kun kaivelin niin tuo kai pitäisi aina olla määritelty eli ehkä tuon null-checkin voi ottaa pois 🤔

Reply:
Okei 👍 On tuossakin sinänsä perusteensa, mietin että ajattelis null arvoa pikemminkin samoin kuin falsea kuten sitä frontendin päässä enemmän käytetään. Mut ehkä vois ottaa poiskin.